src/RulesTest/PropertyConditions/BoolPropertiesTest.cs

```diff
@@ -315,5 +315,36 @@ public void TestIsControlElementDoesNotExistFalse()
                 Assert.IsFalse(IsControlElementDoesNotExist.Matches(e));
             } // using
         }
+
+        [TestMethod]
+        public void IsDialog_true()
```

Comment:
Naming conventions are a bit different in the new methods :(

Reply:
Yeah, I thought about that, but this is the convention we try to use in newer tests. I don't like the mismatch either, but it doesn't bother me enough to change all the tests. Thoughts?

Reply:
Looks like other tests are using upper case after each underscore. If we do that, we get:
```suggestion
        public void IsDialog_True()
```